conf.py

```diff
@@ -26,6 +27,48 @@
 
 htmlhelp_basename = 'behat'
 
+# Files in these paths are copied to the _static output directory e.g. theme/static/img/file.png
+# will become "/_static/img/file.png". They will overwrite any file from the theme with the same
+# name.
+html_static_path = ['_theme/static']
+
+# CSS files here are relative to html_static_path
+html_css_files = ['css/behat-docs.css']
+
+# Files in these paths override templates provided by the theme
+templates_path = ['_theme/templates']
+
 html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "/")
-html_theme_path = ["_themes"]
-html_theme = 'borg'
+html_theme='sphinx_wagtail_theme'
+html_theme_options = dict(
+  # Navbar branding
+  project_name = "Behat",
+  logo = 'img/behat-b-2x-white.png',
+  logo_alt = 'Behat',
+  logo_url = '/',
+  logo_width = 36,
+  logo_height = 50,
+
+  # Base path for "Edit on Github" links
+  github_url="https://github.com/Behat/docs/blob/v3.0/",
+
+  # Global header / footer links as text|target
+  header_links=', '.join([
+    'Guides|/guides.html',
```

Comment:
wouldn't this be broken on RTD as well as the documentation is served on https://docs.behat.org/en/latest/, not https://docs.behat.org/